xbmc/interfaces/legacy/Control.cpp

```diff
@@ -150,7 +150,7 @@ namespace XBMCAddon
     {
       if (!pGUIControl) return NULL;
 
-      LOCKGUI;
+      XBMCAddonUtils::GuiLock(languageHook);
```

Comment:
According to the original PR not:
https://github.com/xbmc/xbmc/pull/12504/files#diff-c7ffaf248f1f66ae2b5477774ff399a3
m_languageHook is a private var inside the class